go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/cert-manager/klone
 
-go 1.21.1
+go 1.23
+
+toolchain go1.23.6
```

Comment:
This line is probably not useful (since we already set the go version to stable in the GH action)?

Reply:
Yeah it's only here because it's automatically added by `go mod tidy`. I'll remove!

Reply:
Removed!